Doc/library/compileall.rst

```diff
@@ -232,7 +232,7 @@ Public functions
    .. versionchanged:: 3.7
       The *invalidation_mode* parameter was added.
 
-.. function:: compile_path(skip_curdir=True, maxlevels=0, force=False, quiet=0, legacy=False, optimize=-1, invalidation_mode=py_compile.PycInvalidationMode.TIMESTAMP)
+.. function:: compile_path(skip_curdir=True, maxlevels=0, force=False, quiet=0, legacy=False, optimize=-1, invalidation_mode=None)
```

Comment:
FWIW, these should probably also contain a `versionchanged` entry as done in https://github.com/python/cpython/commit/a6b3ec5b6d4f6387820fccc570eea08b9615620d#diff-ff4f9cbddd1c0a6371e3cc85fbea0785R80

This should be done for each of the methods whose signature is now changed, mentioning about the change and how default value is determined.

Reply:
Done